@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/ihadss/fn_ihadssDraw.sqf

```diff
@@ -26,7 +26,7 @@ _zoomstate = 0;
 _targhead = 0;
 
 _sensor = "R ";
-_sensxm = "FCR";
+_sensxm = "TADS";
```

Comment:
Changing this shouldn't make a difference, but it shouldn't break anything either.